apidef/oas/schema/x-tyk-gateway.md

```diff
@@ -88,6 +88,9 @@ Versions contains a list of versions that map to individual API IDs.
 **Field: `stripVersioningData` (`boolean`)**
 StripVersioningData is a boolean flag, if set to `true`, the API responses will be stripped of versioning data.
 
+**Field: `fallbackToDefault` (`boolean`)**
+FallbackToDefault controls whether the default API is invoked when the requested version can't be found.
```

Comment:
```suggestion
FallbackToDefault is a boolean flag, if set to `true`, the default API is invoked when the requested version can't be found.
```

If it's good enough for `stripVersioningData`, it's good enough here

Reply:
It starts like `**Field: fallbackToDefault (boolean)**`. Why would we need to say `a boolean` flag again. I think `stripVersioningData` should be changed here.

Reply:
I agree with @furkansenharputlu that this is superfluous. The existing docs [page](https://tyk.io/docs/tyk-apis/tyk-gateway-api/oas/x-tyk-oas-doc/) has a mix of styles for describing Booleans, though most don't have this phrase.

My suggested wording is:

 ```suggestion
FallbackToDefault controls the behaviour of Tyk when a versioned API is called with a nonexistent version number. If set to `true` then the default API version will be invoked; if set to `false` Tyk will return an HTTP 404 "Version does not exist" error in this scenario.
```